spec/parsers/bulkrax/application_parser_spec.rb

```diff
@@ -37,5 +39,40 @@ module Bulkrax
         end
       end
     end
+
+    describe '#base_path' do
+      before do
+        allow(Site).to receive(:instance).and_return(site)
+        allow(Site.instance).to receive(:account).and_return(account)
+      end
+
+      context 'in a hyku enabled app' do
+        before do
+          ENV['SETTINGS__MULTITENANCY__ENABLED'] = 'true'
```

Comment:
would it be worth it to do this test again but with `ENV['HYKU_MULTITENANT'] = 'true'`?

Reply:
I thought about that...but it would end in the same result. plus to me the purpose of the test was to just make sure that if we're in a multi tenant app, that we get the right path and in a non multi tenant app, we get the right path. 
not to check per se all the ways we can get to those results.

Reply:
ah yes agreed then